DEVGUIDE.md

```diff
@@ -275,162 +275,26 @@ Where `<version>` corresponds to the latest Visual Studio version on your machin
 
 Use the `Debug` configuration to test your changes locally. It is the default. Do not use the `Release` configuration! Local development and testing of Visual Studio tooling is not designed for the `Release` configuration.
 
-### Writing and running benchmarks
+### Benchmarking
 
-Existing compiler benchmarks can be found in `tests\benchmarks\`.
+Existing compiler benchmarks can be found in `tests\benchmarks\`. The folder contains READMEs describing specific benchmark projects as well as guidelines for creating new benchmarks. There is also `FSharp.Benchmarks.sln` solution containing all the benchmark project and their dependencies.
+
+To exercise the benchmarking infrastructure locally, run:
+
+```cmd
+build.cmd -configuration $(_BuildConfig) -testBenchmarks
```

Comment:
Needs Linux & macOS versions.

Reply:
Yep, thanks. Don't have a machine to test this but should be just using the shell script instead of cmd, right?

Reply:
Oh, I left additional comment, flag needs to be added to shell script, and double dashes (`--`) should be used, as it's standard linux/Mac convention.

Reply:
Okay, updated all, please check if this works, I don't have a Linux/MacOS device so it's all just blind copypasting + ChatGPT.

Reply:
@psfinaki gotta try WSL for this, man! It's very easy to get a Linux machine set up to start working these days :)

Reply:
Yeah right, I forgot I already have this installed, up and running, hang on :D

Reply:
Okay this should be good to go now

Reply:
![image](https://github.com/dotnet/fsharp/assets/5451366/c9c9867c-64ba-4fd5-8c51-9bb2cdd9944c)